src/frontend/vite.config.ts

```diff
@@ -14,5 +14,14 @@ export default defineConfig({
   build: {
     manifest: true,
     outDir: '../../InvenTree/web/static/web'
+  },
+  server: {
+    proxy: {
+      '/api': {
+        target: 'http://localhost:8000',
+        changeOrigin: true,
+        secure: true
```

Comment:
Question: How does this handle certificates? Ie if the Django server is only available with https

Reply:
Not sure, but this is only for development (I haven't setup https there) and secondly you don't need to use that feature and develop like you already do for now.

Reply:
Sounds good